src/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum Market {
+  L_BTC = '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
+  L_USDT = '9b3d7db7f93cae59e9407232b9864b54c21cba20b992d707fa45136fbfa9b8fa',
+}
+
+export enum CurrencyID {
```

Comment:
is it used somewhere this enum? I see you using `data.json` as a way to hardcode asset hashes already

Reply:
fixed this

Reply:
fixed this

Reply:
fixed this

Reply:
fixed this